common/services/catalogue/urls.js

```diff
@@ -6,7 +6,7 @@ type WorksUrlProps = {|
   page: ?number,
   workType?: ?(string[]),
   itemsLocationsLocationType?: ?(string[]),
-  queryType?: ?string,
+  _queryType?: ?string,
```

Comment:
Would it be better as an enum of the possible query types?

Reply:
I did think this, but if we add new ones in the API, we'd have to keep adding them here. Not that that is a problem.

Reply:
Done.